src/chart/Scatter.js

```diff
@@ -0,0 +1,184 @@
+    Scatter.prototype.publish("pointShape", "cross", "set", "Shape of the data points", ["circle", "rectangle", "cross"]);
+    Scatter.prototype.publish("pointSize", 8, "number", "Point Size");
+
+    Scatter.prototype.updateChart = function (domNode, element, margin, width, height) {
+        var context = this;
+
+        this._palette = this._palette.switch(this.paletteID());
+
+        if (this._prevPointShape !== this.pointSize()) {
+            this.svgData.selectAll(".data").remove();
+            this._prevPointShape = this.pointSize();
+        }
+
+        var column = this.svgData.selectAll(".data")
+            .data(this._data, function (d) { var k = d.filter(function(d, i) {return i > 0;}); return k; })
+        ;
+
+        column.enter().append("g")
+            .attr("class", "data")
+        ;
+
+        column
+            .each(function (d, i) {
+                var element = d3.select(this);
+                var title;
+                var title1;
+                var cx, cy, x, y, x1, y1, x2, y2;
+                var point = element.selectAll(".point").data(d.filter(function(d, i) {return i > 0;}));
+                var point1 = element.selectAll(".point1").data(d.filter(function(d, i) {return i > 0;}));
+                switch (context.pointShape()) {
+                    case "circle" :
+                        title = point
+                          .enter().append("circle")
+                            .attr("class", "point")
+                            .on("click", function (d2, idx) {
+                                context.click(context.rowToObj(d), context._columns[idx + 1]);
+                            })
+                            .append("title")
+                        ;
+                        break;
+
+                    case "rectangle" :
+                        title = point
+                          .enter().append("rect")
+                            .attr("class", "point")
+                            .on("click", function (d2, idx) {
+                                context.click(context.rowToObj(d), context._columns[idx + 1]);
+                            })
+                            .append("title")
+                        ;
+                        break;
+
+                    case "cross" :
+                        title = point
+                          .enter().append("line")
+                            .attr("class", "point")
+                            .attr("stroke-width", 2)
+                            .on("click", function (d2, idx) {
+                                context.click(context.rowToObj(d), context._columns[idx + 1]);
+                            })
+                            .append("title")
+                        ;
+                        title1 = point1
+                            .enter().append("line")
+                            .attr("class", "point1")
+                            .attr("stroke-width", 2)
```

Comment:
Adding this to style will add a stroke on everything else as well, i.e. circles and also the rectangles. In that case, we will have to resize everything, and also handle the strokes on circle and rectangles separately.